src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -220,6 +228,26 @@ module sonic-acl {
 				leaf INNER_L4_DST_PORT {
 					type uint16;
 				}
+
+				leaf VLAN_ID {
+					type uint16 {
+						range 1..4093;
```

Comment:
>4093 [](http://example.com/codeflow?start=15&length=4)

4094 should be good?
How about `0..4095`? the borders are revered but still valid.

Reply:
Updated to 1-4094 which matches the "config vlan" CLI from [sonic-utilities](https://github.com/Azure/sonic-utilities/blob/1ea88e21f72ccbc73e18d9fa5d1223b378000490/utilities_common/cli.py#L246) .  0 is reserved for priority tag and will be assigned PVID which will be in range 1-4094. 4095 is reserved as well .